docs/kraken-x3-z3-guide.md

```diff
@@ -28,6 +28,9 @@ In addition to this, Kraken Z coolers restore the embedded fan controller that i
 
 Kraken 2023 AIOs use the same pump and as their Z3 predecessor but the integrated led controller has been removed. The LCD resolution is 240x240 for the standard version and 640x640 for the elite one.
 
+## NZXT Kraken 2024 Elite RGB
+
+The functionality of the 2024 RGB AIO is identical to the 2023 model, retaining the 640x640 LCD. There's now a light ring on the pump housing. As of the 1.2.0 firmware, are still able to use GIF and static modes. 
```

Comment:
`As of the 1.2.0 firmware, are still able to use GIF and static modes.`

Weird sentence construct? `are` -> `it's` probably.